sdcm/cluster.py

```diff
@@ -4668,7 +4668,7 @@ def node_setup(self, node: BaseNode, verbose: bool = False, timeout: int = 3600)
         if self.params.get('use_mgmt') and self.node_type == "scylla-db":  # pylint: disable=no-member
             self.install_scylla_manager(node)
 
-    def node_startup(self, node: BaseNode, verbose: bool = False, timeout: int = 3600):
+    def node_startup(self, node: BaseNode, verbose: bool = False, timeout: int = 10800):
```

Comment:
Probably need to specify this increased timeout in call place based on the dc number.
Changing this default